core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java

```diff
@@ -417,7 +418,10 @@ public void updateForLocation(String pid, String location,
         // sanity check to make sure no values are
         // null
         for (Entry<String, Object> curEntry : configurationTable.entrySet()) {
-            if (curEntry.getValue() == null) {
+            Object value = curEntry.getValue();
+            if (value instanceof JSONArray && ((JSONArray)value).size() == 0) {
```

Comment:
JSONArray extends ArrayList, so this whole if block could be replaced with CollecitonUtils.isEmpty().